docs/conf.py

```diff
@@ -47,16 +47,16 @@
 # General information about the project.
 project = u'UnifyFS'
 copyright = u'2020, Lawrence Livermore National Security LLC, LLNL-CODE-741539, UT-Batelle LLC'
-author = u'Kathryn Mohror, Adam Moody, Oral Sarp, Feiyi Wang, Hyogi Sim, Swen Boehm, Michael Brim, Danielle Sikich, Joseph Moore, Ned Bass, Tony Hutter, Celso Mendes, Craig Steffen, Cameron Stanavige'
+author = u'Kathryn Mohror, Adam Moody, Oral Sarp, Feiyi Wang, Hyogi Sim, Swen Boehm, Michael Brim, Danielle Sikich, Joseph Moore, Ned Bass, Tony Hutter, Celso Mendes, Craig Steffen, Hariharan Devarajan, Cameron Stanavige'
```

Comment:
It's 'Sarp Oral', not 'Oral Sarp'

Reply:
Nice catch. Wonder how long it's been like that.